docs/source/quickstart.rst

```diff
@@ -52,27 +48,26 @@ Dependencies
   interactive plots. As an alternative to PyQt, Chaco is being tested more and
   more with the `PySide <http://www.pyside.org/>`_ toolkit (LGPL license).
 
-.. .. note
-.. ::
-.. In addition to wxPython or PyQt a cross-platform OpenGL backend (using
-.. Pyglet) is in the works, and it will not require WX or Qt.
+.. Note::
+   In addition to wxPython or PyQt a cross-platform OpenGL backend (using
+   Pyglet) is in the works, and it will not require WX or Qt.
```

Comment:
I would actually just remove this note. I don't forsee _increased_ support for Pyglet in Enable.